test/cmd/builds.sh

```diff
@@ -12,16 +12,64 @@ os::log::install_errexit
 url=":${API_PORT:-8443}"
 project="$(oc project -q)"
 
+# expect_build_output_spec runs `oc new-build` with given arguments and expects
+# a build config with a given name to be created with a given build output spec
+function os::cmd::builds::expect_build_output_spec() {
```

Comment:
Testing cli invocations is (somewhat) reasonably done via bash. We already had a bash framework, we're just trying to make it (more) reasonable to jump in and know exactly what is supposed to be getting checked.